apps-rendering/package.json

```diff
@@ -37,7 +37,7 @@
     "@creditkarma/thrift-server-core": "^1.0.4",
     "@emotion/jest": "^11.10.0",
     "@guardian/apps-rendering-api-models": "^1.1.1",
-    "@guardian/atoms-rendering": "^23.6.0",
+    "@guardian/atoms-rendering": "^23.7.2",
```

Comment:
Atoms [should also be handling](https://github.com/guardian/atoms-rendering/pull/471) the SpecialReportAlt theme.